pkgs/development/python-modules/emv/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, click, enum-compat, pyscard, pycountry, terminaltables
+, pytestCheckHook, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "emv";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "russss";
+    repo = "python-emv";
+    rev = "v${version}";
+    hash = "sha256:1ldibcy1wgnaimf8ri79lg8im1mhfqajrigfflvizw0hb0ivh8al";
+  };
+
+  checkInputs = [ pytestCheckHook unittest2 ];
```

unittest2 should only be needed in python2, also, upstream only has a pytest as a non-linting utility https://github.com/russss/python-emv/blob/main/dev-requirements.txt
```suggestion
  checkInputs = [ pytestCheckHook ];
```

---

Yeah; I was pinned to 1.0.9 because that was the latest version I'd actually tested. 1.0.11 has some fixes I contributed and has now been released so I've updated to that, but it depends on pyscard 2.0.0 which isn't packaged in nixpkgs yet.